### DISCUSSION TITLE
Automatic Re-order edges.

### DISCUSSION THREAD

**Discussion by nkeisel-gw** [xyflow/xyflow#1842] [Ideas]
## Description
We are using `react-flow` and its `updatable edges` feature to allow users to be able to reconnect existing edges - move an edge from one node to another.

## Scenario
- User added `edge-2` (`Node A` ---> `Node C`) and `edge-3` (`Node B` ---> `Node C`), so `Node C` has 2 edges connected to.
- We want to move/update the edges.

![image](https://user-images.githubusercontent.com/97993600/150034074-a6ab7147-cb9b-4819-8bb9-c7003adf970a.png)

## Problem
- We can move/update `edge-3` easily, because it was added lastly, and is positioned on top of all the other edges, so it has top-priority when user tries to move among the other edges overlapping.
- We can't move/update `edge-2`, because it is overlapped with `edge-3`, and it was created before `edge-3`, so it is positioned behind `edge-3`, so when you try to select/move/update `edge-2`, it always focus `edge-3`, and it is very difficult to update `edge-2`.

## What we want to do?
We have all the edges selectable, so each edge is highlighted when it is selected, so we can always manage the selected edge.
**Is it possible to re-position the selected edge to be on top of all the other edges, so that it can be easily selected/movable/updatable?**

Thanks

**Comment by chikamichi** [comment]
I guess with v10 you will be able to edit an edge’s z-index upon clicking on it, allowing the user to then grab it.

See https://github.com/wbkd/react-flow/pull/1555

**Reply by nkeisel-gw** [reply]
@chikamichi Thanks for your reply. Do you also know when v10 would be released?

**Reply by moklick** [reply]
Mid March!

**Reply by visualsitemaps** [reply]
any updates on this?
